pkgs/shells/nushell/plugins/hcl.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  IOKit,
+  CoreFoundation,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nu_plugin_hcl";
+  version = "0.100.0";
```

Comment:
~This is the version of nushell, not of the plugin. The plugin had 0.88.1 tag released in Dec 2023, so that would be the base version, and the specific commit was released yesterday, so the correct version for this is
`0.88.1-unstable-2024-11-22` (<last version release>-unstable-<date of commit> for commits that are not version releases)~ Nevermind, I see that the Cargo.toml version was updated to 0.100.0 in that commit, so I think this version number is correct (AFAIK it can be based on metadata in the repo like that and not just git tags).

Reply:
I try to keep the version number in sync with nushell's own so that it's immediately obvious whether given plugin version is compatible with current nushell